kalite/topic_tools/__init__.py

```diff
@@ -123,12 +124,27 @@ def get_assessment_item_cache(force=False):
 
     return ASSESSMENT_ITEMS
 
+def recurse_topic_tree_to_create_hierarchy(node, level_cache={}, hierarchy=[]):
```

Comment:
Khan-specific topic tree hierarchy generation for coach report topics. I don't think these get used anywhere else now. Hopefully we can blow this away with the coach reports redesign.